kitsune/wiki/config.py

```diff
@@ -28,6 +28,20 @@
         u'como-fazer-o-firefox-parar-de-se-conectar-automati',
     ]
 }
+FALLBACK_LOCALES = {
+    'bn-IN': ['bn-BD'],  # Bug 800880
+    'ca': ['es'],  # Bug 800880
+    'eu': ['es'],  # Bug 800880
+    'gl': ['es'],  # Bug 800880
+    'gu-IN': ['hi-IN'],  # Bug 800880
+    'wo': ['fr'],  # Bug 800880
+    'fy-NL': ['nl'],  # Bug 800880
+    'wo': ['fr'],  # Bug 800880
+    'pt-PT': ['pt-BR'],  # Bug 800880
+    'ta-LK': ['ta'],  # Bug 800880
+    'zh-CN': ['zh-TW'],  # Bug 800880
```

Comment:
Please change to:

'zh-HK': ['zh-TW'],  # Bug 800880

as per Peter's comment (https://bugzilla.mozilla.org/show_bug.cgi?id=800880#c27)